elfeed-org.el

```diff
@@ -36,6 +36,8 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'cl-lib)
+(require 'xml)
```

Comment:
I remembered, `Package-Requires` is not necessary at all, melpa will fill it automatically 😂